[esx_addons]/esx_banking/fxmanifest.lua

```diff
@@ -1,7 +1,9 @@
-fx_version 'adamant'
+fx_version 'cerulean'
 
 game 'gta5'
 
+author 'Gellipapa && Raven'
```

Comment:
We can accept the ## Special Thanks in the README, but this author field must either be empty or say `author 'ESX Framework'` as any PR that is accepted will fall under the ESX License.